### ISSUE TITLE
Sending images

### ISSUE THREAD

**Issue by chrishardin** [paulpfeister/GroupMe-BotDriver#2]
I’m trying to get the bot to send an image but I can’t exactly get it to work. I thought that maybe i could use attach_type(), but not really sure how to use it. 

TY

**Comment by ZacharyMass** [comment]
I agree, trying to figure out the same issue, and while it appears that the attachment of images is implemented through some functions, I don't see it implemented into the send_messages function which would be needed for it to work.  Further clarification would be helpful.

**Comment by ZacharyMass** [comment]
I just submitted a pull request that would make this happen.  @chrishardin if you want to use that code, it should work fine for you.

**Comment by paulpfeister** [comment]
Sorry I'm late! Life gets away from you sometimes, and I haven't been able to maintain this as much as I'd like.

@ZacharyMass The PR looks simple enough where there shouldn't be any issues, but just to be safe I'm going to test it out before merging sometime this week.

@chrishardin How has his addition treated you?

**Comment by gansel51** [comment]
@paulpfeister @ZacharyMass I've been using this in my groupme bot, which is based on Paul's. However, this fix doesn't work for mobile users. Images appear properly for web users, but appear with a gray box and the link to the photo on the app. Haven't had time to work on a fix but figured I'd pass it along!
